### DISCUSSION TITLE
ETA distribution transformations

### DISCUSSION THREAD

**Discussion by EmanIbrahim137** [nlmixrdevelopment/nlmixr#593] [Q&A] [answered]
Hello,

I am wondering how I can apply transformations of ETA distribution as box-cox in nlmixr.

Also, is there a way to use Mixture models?

thanks,
Eman

**Comment by mattfidler** [comment, marked as answer]
I would suggest using the same method DOI: [10.1007/s11095-009-9931-1](https://doi.org/10.1007/s11095-009-9931-1)

There is no way to use mixture models at this time.

**Reply by mattfidler** [reply]
Note for cox box this would be:

```r
etaBox = (exp(eta.est)*thetaBox - 1)/thetaBox
```
